### ISSUE TITLE
radiometric offsets L1C

### ISSUE THREAD

**Issue by safalabolo** [Evoland-Land-Monitoring-Evolution/sentinel2_superresolution#7]
A simple question: are the Sentinel-2 L1C bands, before applying super resolution, corrected with "radiometric offsets" (https://scihub.copernicus.eu/news/News00931)?

**Comment by jmichel-otb** [comment]
No the radiometric offsets are not applied beforehand. Feel free to apply them on super-resolved products.

**Comment by safalabolo** [comment]
@jmichel-otb 
I also imagine when talking about the L2A (theia) level, is that correct?

**Comment by jmichel-otb** [comment]
I think in this case the offset is handled in MAJA (the L1C -> L2A processor).

I updated the FAQ.

**Comment by eomasters-repos** [comment]
I've also stumbled upon this offset. Compring the read data did not provide the same values as reading the data with other tools.
I think the radiometric offset of -1000 should actually not be applied. This offset is applied by the [sensorsio](https://github.com/CNES/sensorsio) library, I've raised an [issue](https://github.com/CNES/sensorsio/issues/1) there.
Also, the no-data value is not correctly considered by sensorsio.
